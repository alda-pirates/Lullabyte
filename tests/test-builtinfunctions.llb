void main(){
	int[] one;
	int[] two;
	int[] three;

	sound a;
	pitch b;
	sound c;

	a = |A4, A5, B6, B7|:0.5:100;
	b = B5;
	c = |C2|:5.0:1;

	one = [1];
	two = [1, 2];
	three = [1, 2, 3];

	print(getPitch(a));
	print(getPitch(b));

	print(getDuration(a));
	print(getDuration(c));

	print(getAmplitude(a));
	print(getAmplitude(c));
	
	print(length(one));
	print(length(two));
	print(length(three));
	print(length([]));

	/* print(length(1)); length can only be called on arrays*/
}