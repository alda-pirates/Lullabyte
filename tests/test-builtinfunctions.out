A4
B5
0.5
5.
100
1
1
2
3
0
